build-system/tasks/validator.js

```diff
@@ -30,6 +30,30 @@ async function validator() {
   execOrDie('cd validator && python build.py' + validatorArgs);
 }
 
+/**
+ * Simple wrapper around the Java validator test suite.
+ */
+async function validatorJava() {
+  const commands = ['cd validator/java'];
```

Comment:
This can be passed in to `exec` via the [`cwd` option](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options).

(I realize that you followed the precedent set by the above tasks. I think they too can use the `cwd` option.)

Reply:
Done (incl. other tasks)